lms/djangoapps/course_home_api/outline/v1/tests/test_views.py

```diff
@@ -97,7 +100,18 @@ def test_handouts(self, is_enrolled, is_staff, is_public, handouts_visible):
         handouts_html = self.client.get(self.url).data['handouts_html']
         self.assertEqual(handouts_html, '<p>Hi</p>' if handouts_visible else '')
 
-    # TODO: write test_get_unknown_course when more data is pulled into the Outline Tab API
+    def test_get_unknown_course(self):
+        url = reverse('course-home-outline-tab', args=['course-v1:unknown+course+2T2020'])
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 404)
+
+    @COURSE_HOME_MICROFRONTEND.override(active=True)
```

Comment:
Don't think you need this since it's already set to True at the class level.

Reply:
🙃